From 79ed2bf657eca22cf9fd3a206e9efaf4a4abf79c Mon Sep 17 00:00:00 2001
From: "Matthew Horsfall (alh)" <wolfsage@gmail.com>
Date: Thu, 8 Jan 2015 08:41:24 -0500
Subject: Fix apidocs for OP_TYPE_IS(_OR_WAS) - arguments separated by |, not
 ,.

This was causing Devel::PPPort's tooling some grief.
(cherry picked from commit 5bfb0af0b8971c7f063804d149384ca17bfc0ae2)

Bug-Debian: https://bugs.debian.org/822336
Patch-Name: fixes/5.20.3/docs/op.diff
---
 op.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/op.h b/op.h
index 9d9dd58d8..ff45354ae 100644
--- a/op.h
+++ b/op.h
@@ -1005,7 +1005,7 @@ For custom ops the type is returned from the registration, and it is up
 to the registree to ensure it is accurate.  The value returned will be
 one of the OA_* constants from op.h.
 
-=for apidoc Am|bool|OP_TYPE_IS|OP *o, Optype type
+=for apidoc Am|bool|OP_TYPE_IS|OP *o|Optype type
 Returns true if the given OP is not a NULL pointer
 and if it is of the given type.
 
@@ -1013,7 +1013,7 @@ The negation of this macro, C<OP_TYPE_ISNT> is also available
 as well as C<OP_TYPE_IS_NN> and C<OP_TYPE_ISNT_NN> which elide
 the NULL pointer check.
 
-=for apidoc Am|bool|OP_TYPE_IS_OR_WAS|OP *o, Optype type
+=for apidoc Am|bool|OP_TYPE_IS_OR_WAS|OP *o|Optype type
 Returns true if the given OP is not a NULL pointer and
 if it is of the given type or used to be before being
 replaced by an OP of type OP_NULL.
